### ISSUE TITLE
Consuming SOAP web services throws errors on Adobe image after 3.4.0

### ISSUE THREAD

**Issue by joshuacurtiss** [Ortus-Solutions/docker-commandbox#63]
While using the `adobe2018` tag, I noticed I could not consume a SOAP web service. It would throw a nasty error message (see below). I was using arm64 but then tested on Intel and had same experience. I kept going back in versions and noticed that my test was finally successful going back to tag `arm64-adobe2018-3.4.0`.

### Reproducing

```bash
docker run --rm -p 8080:8080 -v ~/Sites/test/app:/app ortussolutions/commandbox:adobe2018
```

~/Sites/test/app/index.cfm:
```cfml
<cfset ws=createObject("webservice","http://127.0.0.1:8080/ws.cfc?wsdl") />
<cfdump var="#ws.test()#" />
```

~/Sites/test/app/ws.cfc:
```
component {
    remote struct function test() {
        return {"id": "test", "name": "Test Test"};
    }
}
```

Resulting error: 
```
coldfusion.jsp.CompilationFailedException: Errors reported by Java compiler: An exception has occurred in the compiler (1.8.0_102). Please file a bug against the Java compiler via the Java bug reporting page (http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com) for duplicates. Include your program and the following diagnostic in your report. Thank you. java.lang.NullPointerException at com.sun.tools.javac.file.Locations.getPathEntries(Locations.java:149) at com.sun.tools.javac.file.Locations.getPathEntries(Locations.java:134) at com.sun.tools.javac.file.Locations.access$000(Locations.java:71) at com.sun.tools.javac.file.Locations$BootClassPathLocationHandler.computePath(Locations.java:617) at com.sun.tools.javac.file.Locations$BootClassPathLocationHandler.lazy(Locations.java:643) at com.sun.tools.javac.file.Locations$BootClassPathLocationHandler.getLocation(Locations.java:577) at com.sun.tools.javac.file.Locations.getLocation(Locations.java:678) at com.sun.tools.javac.file.JavacFileManager.getLocation(JavacFileManager.java:803) at com.sun.tools.ja....
--
 
The error occurred in /app/index.cfm: line 1
1 : <cfset ws=createObject("webservice","http://127.0.0.1:8080/ws.cfc?wsdl") /> 
2 : <cfdump var="#ws.test()#" />
```

Thank you very much for your good work with these images.

**Comment by jclausen** [comment]
@joshuacurtiss This is a legit issue with 2018 and 2021 when using webservices.  The issue is a known issue with ACF because the webservice implementation uses a `tools.jar` file ( https://tracker.adobe.com/#/view/CF-4206461 ) which is only present in the JDK, but not the JRE.  We build with the JRE because this saves us about 140MB in image size.  

We are going to look in to some solutions, but I have also opened a bug with Adobe. Hopefully, they will de-couple the need for a full JDK installation in the near future. https://tracker.adobe.com/#/view/CF-4215554

**Comment by jclausen** [comment]
@joshuacurtiss Update:  I am changing the Adobe image builds over to use a full JDK installation.  This means that there will be an extra 150MB, but at least full functionality for webservices will be there.  This should be fixed in version 3.6.2 of the image.
